### PR TITLE
Update function error references

### PR DESCRIPTION
**Pull request by ElijahSwiftIBM**
-Change SecurityRequestError references within documented functions to correct misleading RC information -Add references to DownstreamFatalError in all functions capable of returning SecurityRequestError

### :bulb: Issue Reference

**Issue:**  #61 

### :computer: What does this address?

Documentation error in 1.0b3. Numerous functions document SecurityRequestError as

Raises SecurityRequestErrorwhen the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to0``

This needs to be updated and all functions capable of raising SecurityRequestError in this way can also raise DownstreamFatalError as well.

### :pager: Implementation Details

N/A

### :clipboard: Is there a test case?

N/A

### CODE REVIEW

File: resource/abstractions/auditing_rules.md [hunk 1]
```diff
@@ -0,0 +1,474 @@
+---
```
Comment (lcarcaramo) [inline resource/abstractions/auditing_rules.md:1]: The Auditing rules documentation shouldn't be included in this PR. These updates are part of #52

Comment (ElijahSwiftIBM) [reply resource/abstractions/auditing_rules.md:1]: Correct, not sure exactly how this happened but it's been removed.

File: user/abstractions/operations.md [hunk 2]
```diff
@@ -29,7 +29,9 @@ Check if a user has **Operations** authority.
 
 #### ❌ Raises
 * `SecurityRequestError`<br>
-  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is equal to `4`.
+* `DownstreamFatalError`<br>
+  Raises `DownstreamFatalError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is greater than `4`.
 
 #### 💻 Example
```
Comment (lcarcaramo) [inline user/abstractions/operations.md:37]: The exceptions for `take_away_operations_authority()` still needs to be updated.

File: access/base/delete.md [hunk 3]
```diff
@@ -44,7 +44,9 @@ Delete a specified **permission**
 
 #### ❌ Raises
 * `SecurityRequestError`<br>
-  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is equal to `4`.
```
Comment (lcarcaramo) [inline access/base/delete.md:47]: I think we can get rid of the `returned by IRRSMO00 part` to reduce noise. I think the following gets the message across:

Also, for clarity, I think we should be referring to the `SAF Return Code`.

```markdown
Raises `SecurityRequestError` when the **SAF Return Code** of a **Security Result** is equal to `4`.
```

File: user/abstractions/operations.md (hunk 2 above)
Comment (ElijahSwiftIBM) [reply user/abstractions/operations.md:37]: Not sure how `operations.md` got missed in my initial crawl but there were two places that needed a touch up. Both addressed.

File: access/base/delete.md (hunk 3 above)
Comment (lcarcaramo) [reply access/base/delete.md]: This issue has been resolved.
